incubator/mysqlha/values.yaml

```diff
@@ -39,6 +39,16 @@ persistence:
   accessMode: ReadWriteOnce
   size: 10Gi
 
+metrics:
+  enabled: false
+  image: prom/mysqld-exporter
+  imageTag: v0.10.0
+  imagePullPolicy: IfNotPresent
+  resources: {}
+  annotations:
```

Comment:
Looks like you missed updating the templates to use the annotations.

Reply:
Good catch, thanks! Updated template accordingly.